LSP.sublime-settings

```diff
@@ -41,6 +41,13 @@
   // hint: 4
   "auto_show_diagnostics_panel_level": 2,
 
+  // Only show diagnostics in the panel with a severity equal to or less than:
+  // error: 1
+  // warning: 2
+  // info: 3
+  // hint: 4
+  "diagnostics_panel_include_severity_level": 4,
```

Comment:
I think it's better if this setting were placed just under "show_diagnostics_severity_level".

Reply:
Moreover, clarify in "show_diagnostics_severity_level" that it only applies to the squiggly lines (and the gutter icons).

Reply:
Done.